### ISSUE TITLE
StringIndexOutOfBoundsException caused by geckodriver

### ISSUE THREAD

**Issue by airudah2000** [bonigarcia/webdrivermanager#43]
There's a StringIndexOutOfBoundsException while attempting to retrieve the geckodriver:

```
2016-07-01 11:08:57:330 [ScalaTest-run] DEBUG BrowserManager:65 - Connecting to https://api.github.com/repos/mozilla/geckodriver/releases to check latest MarionetteDriver release
2016-07-01 11:08:57:330 [ScalaTest-run] TRACE BrowserManager:348 - Checking the lastest version of [wires, geckodriver]
2016-07-01 11:08:57:331 [ScalaTest-run] TRACE BrowserManager:349 - Input URL list [https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-arm7hf.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-linux64.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-mac.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-win64.zip, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0.tar.gz]
2016-07-01 11:08:57:331 [ScalaTest-run] TRACE BrowserManager:355 - URL https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0.tar.gz match with geckodriver
*** RUN ABORTED *** (2 seconds, 933 milliseconds)
  java.lang.RuntimeException: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
  at io.github.bonigarcia.wdm.BrowserManager.manage(BrowserManager.java:272)
  at io.github.bonigarcia.wdm.BrowserManager.setup(BrowserManager.java:103)
  at io.github.bonigarcia.wdm.BrowserManager.setup(BrowserManager.java:82)
  at safe.SafeBaseSpec.<init>(SafeBaseSpec.scala:87)
  at safe.A_BeforeTestsSpec.<init>(A_BeforeTestsSpec.scala:10)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
  at java.lang.Class.newInstance(Class.java:442)
  at org.scalatest.tools.Runner$.genSuiteConfig(Runner.scala:2644)
  at org.scalatest.tools.Runner$$anonfun$37.apply(Runner.scala:2461)
  at org.scalatest.tools.Runner$$anonfun$37.apply(Runner.scala:2460)
  at scala.collection.immutable.List.map(List.scala:273)
  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:2460)
  at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:1044)
  at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:1043)
  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:2722)
  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1043)
  at org.scalatest.tools.Runner$.run(Runner.scala:883)
```

Here are the versions we are using:

```
<dependency>
    <groupId>org.seleniumhq.selenium</groupId>
    <artifactId>selenium-java</artifactId>
    <version>2.53.1</version>
</dependency>
<dependency>
    <groupId>io.github.bonigarcia</groupId>
    <artifactId>webdrivermanager</artifactId>
    <version>1.4.5</version>
</dependency>
```
 
This has just started today. We traced it down to the last url in the list below where we noticed that the last url has just one hyphen. 
The code expects 2 : (at BrowserManager.java getLatest() method)
`2016-07-01 10:51:29:867 [ScalaTest-run] TRACE BrowserManager:349 - Input URL list [https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-arm7hf.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-linux64.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-mac.tar.gz, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-win64.zip, https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0.tar.gz]
`

**Comment by bonigarcia** [comment]
Thanks for reporting. I'll look into this ASAP.

**Comment by airudah2000** [comment]
Thanks, 

We have just noticed they released 18 hours ago.

https://github.com/mozilla/geckodriver/releases/ 

It seems that they now have a "all" release without a second hyphen. We raised this.
https://github.com/mozilla/geckodriver/issues/133

**Comment by bonigarcia** [comment]
Now the source tar ball has been removed by geckodriver developers. Nevertheless there is still a problem related with the version name. I am going to solve it and release a new version of WebDriverManager.

**Comment by airudah2000** [comment]
Awesome. I'll be watching

**Comment by bonigarcia** [comment]
I've just released WebDriverManager 1.4.6. It will be available on Maven Central in a few minutes.
